### DISCUSSION TITLE
Adicionar contas

### DISCUSSION THREAD

**Discussion by jooilson** [RamonSilva20/mapos#1585] [Ideas]
Seria interessante ter a opção de adicionar contas para lançar direto nela (ex: cliente pagou em pix pelo BB e outro pelo MercadoPago), controlando de maneira mais eficiente a assistência.

**Comment by SuperPorte** [comment]
Verdade.
Sugestão, por no faturamento, um campo valor. Assim daria para ir adicionando linhas com os valores e formas de pagamento

**Comment by jooilson** [comment]
Estou interessado em dar andamento nessa ideia. Alguém tem alguma dica ou sugestão?
